src/auth/models/strategy.class.ts

```diff
@@ -182,6 +186,7 @@ export abstract class Strategy extends events.EventEmitter {
         this.initializePassport()
         this.initializeSession()
         this.initializeKeepAlive()
+        this.initialiseCSRF()
```

Comment:
```suggestion
 if (this.options.useCSRF) {
        this.initialiseCSRF()
        }
```

Reply:
that is already part of the initialise function:
public initialiseCSRF = (): void => {
        if (this.options.useCSRF) {
            this.logger.log('initialising CSRF middleware')
            this.router.use(csrf())
        }
    }

Reply:
I know but inside the method initialiseCSRF. Inside vs outside was my question